addons/spreadsheet_dashboard/tests/test_dashboard_data.py

```diff
@@ -12,3 +15,32 @@ def test_validate_dashboard_data(self):
         for dashboard in dashboards:
             with self.subTest(dashboard.name):
                 self.validate_spreadsheet_data(dashboard.raw, dashboard.name)
+
+    @mute_logger('odoo.sql_db')
+    def test_onchange_data_valid_json(self):
```

Comment:
could you move those tests in another test class which is not `post_install` ? thanks :)